plugins/plugin-chart-echarts/src/BoxPlot/controlPanel.ts

```diff
@@ -0,0 +1,114 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { t, validateNonEmpty } from '@superset-ui/core';
+import { D3_FORMAT_OPTIONS, formatSelectOptions } from '@superset-ui/chart-controls';
+
+export default {
+  controlPanelSections: [
+    {
+      label: t('Query'),
+      expanded: true,
+      controlSetRows: [
+        ['metrics'],
+        ['adhoc_filters'],
+        ['groupby'],
+        ['columns'],
+        ['limit'],
+        [
+          {
+            name: 'whiskerOptions',
+            config: {
+              type: 'SelectControl',
+              freeForm: true,
+              label: t('Whisker/outlier options'),
+              default: 'Tukey',
+              description: t('Determines how whiskers and outliers are calculated.'),
+              choices: formatSelectOptions([
```

Comment:
`renderTrigger: true`, maybe?

Reply:
The whiskers actually affect what's returned in the data payload, so unfortunately `renderTrigger: true` won't work here.